### PR TITLE
Update device-list.md

### PR DESCRIPTION
**Pull request by rekhanr**
1) Moved Dell 3530 to the beginning of the list. 2) Added a row for 7200 2-in-1/7300/7400

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/service-description/device-list.md [hunk 1]
```diff
@@ -28,10 +28,11 @@ The links to devices here are for your reference only. If you want to order devi
...
 |[Dell Latitude 5300/5400/5500](https://www.dell.com/en-us/work/shop/dell-laptops-and-notebooks/latitude-5400-business-laptop/spd/latitude-14-5400-laptop) | 256 GB / Intel i5 / 8 GB RAM | IR camera required | June 1, 2021  | June 1, 2024  |
 [Dell Latitude 5490](https://www.dell.com/en-us/work/shop/cty/pdp/spd/latitude-14-5490-laptop?cid=265768&st=dell%2Blatitude%2B5490&VEN1=yKFOheND,73873544881479,901q5c14135,c,,%7BProductid%7D&VEN2=be,dell%2Blatitude%2B5490&lid=5980636&dgc=st&dgseg=so&acd=12309152537461020&VEN3=112504543746142297) | 256 GB / Intel i5 / 8 GB RAM | IR camera required | Jan 9, 2020  | Jan 9, 2023  |
+[Dell Latitude 7200 2-in-1/7300/7400](https://www.dell.com/en-us/work/shop/dell-laptops-and-notebooks/latitude-7200-2-in-1-business-laptop/spd/latitude-12-7200-2-in-1-laptop) | 256 GB / Intel i5 / 8 GB RAM | IR camera required | May 1, 2021  | May 1, 2024  |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/device-list.md]:
```suggestion
[Dell Latitude 7200 2-in-1  / 7300 / 7400](https://www.dell.com/en-us/work/shop/dell-laptops-and-notebooks/latitude-7200-2-in-1-business-laptop/spd/latitude-12-7200-2-in-1-laptop) | 256 GB / Intel i5 / 8 GB RAM | IR camera required | May 1, 2021  | May 1, 2024  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@jaimeo - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**